login.html

```diff
@@ -1,63 +1,46 @@
 <!DOCTYPE html>
 <html>
 <head>
-	<meta charset="UTF-8">
+    <meta charset="UTF-8">
 	<!-- mobile support -->
 	<meta name="viewport" content="width=device-width, initial-scale=1">
 	<title>UWCSSA Membership Management System - Find Member</title>
 	<link rel="stylesheet" href="css/bootstrap.css">
 	<link rel="shortcut icon" href="img/icon.jpg">
+	<link rel="stylesheet" href = "css/style.css">
 </head>
 <body>
 	<!-- top navigation bar -->
-	<div class="navbar navbar-default navbar-static-top" role="navigation">
-		<div class="container">
-			<!-- Brand and toggle get grouped for better mobile display -->
-			<div class="navbar-header">
-				<a href="index.html" class="navbar-brand">UWCSSA Membership Management System</a>
-				<button class="navbar-toggle" type="button" data-toggle="collapse" data-target="#navbar-main">
-					<span class="icon-bar"></span>
-					<span class="icon-bar"></span>
-					<span class="icon-bar"></span>
-				</button>
-			</div>
-			<div class="navbar-collapse collapse" id="navbar-main">
-				<ul class="nav navbar-nav">
-					<li class="active">
-						<a href="index.html">Register</a>
-					</li>
-				</ul>
-				<ul class="nav navbar-nav navbar-right">
-					<li class="active"><img src="img/logo.png" width="110" height="50"></li>
-				</ul>
-			</div>
-		</div>
-	</div>
+
```

Comment:
I think it's ok to keep the Navbar, a plaint login page with only email and password fields looks kind wired and informal.